src/components/HistoryTables/types.ts

```diff
@@ -15,7 +15,7 @@ export type MessageConfirmedRow = Pick<
 
 export type MessagePendingRow = Pick<
   MessagePending,
-  'cid' | 'method' | 'from' | 'to' | 'value'
+  'cid' | 'method' | 'height' | 'from' | 'to' | 'value'
```

Comment:
This change doesn't really make sense to me - a pending message doesn't have a height because that would imply it's confirmed

Reply:
That is interesting, according to the schema it does, as a string (see my comment about the height types above).

Anyway, it's fine to remove, but then we have to adjust the prop types passed to `useAge` a bit to make typescript happy, see my latest commit

Reply:
Just did a test and height seems to return an empty string for PendingMessage

Reply:
https://github.com/glifio/graph/issues/67